app/sidekiq/form1010cg/submission_job.rb

```diff
@@ -39,13 +52,14 @@ def perform(claim_id)
       log_exception_to_sentry(e)
       StatsD.increment("#{STATSD_KEY_PREFIX}retries")
 
-      increment_applications_retried(claim_id)
+      increment_applications_retried(claim_id) unless Flipper.enabled?(:caregiver1010)
 
       raise
     end
 
     private
 
+    # TODO: @coope93 to remove increment method and feature (:caregiver1010) after validating functionality
```

Comment:
Ideally there's a date attached (<90 days)

Reply:
+1.
maybe even a link to a ticket for the work?